browser/src/map/handler/Map.Print.js

```diff
@@ -39,6 +39,7 @@ L.Map.Print = L.Handler.extend({
 		var blob = new Blob([e.response], {type: 'application/pdf'});
 		var url = URL.createObjectURL(blob);
 		this._printIframe = L.DomUtil.create('iframe', '', document.body);
+		this._printIframe.title = 'Print dialog';
```

Comment:
All the strings which are available for the user should be translated.
`'Print dialog'` -> `_('Print dialog')`